code/datums/components/footstep.dm

```diff
@@ -88,6 +88,9 @@
 			var/mob/living/carbon/human/H = LM
 			var/feetCover = (H.wear_suit && (H.wear_suit.body_parts_covered & FEET)) || (H.w_uniform && (H.w_uniform.body_parts_covered & FEET))
 			
+			if(H.dna.species.special_walk_sounds)
+				playsound(T, pick(H.dna.species.special_walk_sounds), 25, TRUE)
```

Comment:
```suggestion
				playsound(T, pick(H.dna.species.special_walk_sounds), 25, TRUE)
				return
```

Reply:
Playing both sounds was intentional, but if you would prefer otherwise I can change that

Reply:
gee billy, mom gave you TWO walk sounds?